spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/discovery/ConditionalOnKubernetesDiscoveryEnabled.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019-2019 the original author or authors.
```

Comment:
moved `ConditionalOnKubernetesDiscoveryEnabled` to the commons package, since it was the same exact annotation in both code bases of the clients